docs/LunaNodeWebDeployment.md

```diff
@@ -2,9 +2,11 @@
 
 This article explains the **deployment of BTCPay Server through LunaNode Web-Wizard**. [LunaNode](http://lunanode.com/) is Canadian based hosting provider that accept Bitcoin payments and requires no personal identification except for a phone number verification.
 
-Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you do not have any technical knowledge. For the price of around US$ 8.8 per month, you can have a **self-hosted BTCpay**, which includes a Bitcoin full node and Lightning Network node.
+Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. However, if you set up a custom domain you will need to have some comfort with the shell.
```

Comment:
```suggestion
Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. However, if you set up a custom domain you will need to have some comfort with the command line interface.
```

Reply:
Just making CLI generic instead of Shell specific. 
An unfamiliar user wouldn't know what shell is.